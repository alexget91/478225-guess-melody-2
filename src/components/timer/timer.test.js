import React from "react";
import renderer from "react-test-renderer";
import Timer from "./timer";

it(`Timer correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<Timer time={300} onTimeChange={jest.fn()} onTimeIsUp={jest.fn()}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
