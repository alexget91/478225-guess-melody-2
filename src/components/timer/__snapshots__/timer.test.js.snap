// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer correctly renders after relaunch 1`] = `
Array [
  <svg
    className="timer"
    viewBox="0 0 780 780"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="timer__line"
      cx="390"
      cy="390"
      r="370"
      style={
        Object {
          "filter": "url(#blur); transform: rotate(-90deg) scaleY(-1); transform-origin: center",
        }
      }
    />
  </svg>,
  <div
    className="timer__value"
    xmlns="http://www.w3.org/1999/xhtml"
  >
    <span
      className="timer__mins"
    >
      05
    </span>
    <span
      className="timer__dots"
    >
      :
    </span>
    <span
      className="timer__secs"
    >
      00
    </span>
  </div>,
]
`;
